spec/operatoroverloading.dd

```diff
@@ -574,8 +574,8 @@ $(H3 $(LNAME2 index_assignment_operator, Index Assignment Operator Overloading))
 	$(P If the left hand side of an assignment is an index operation
 	on a struct or class instance,
 	it can be overloaded by providing an $(D opIndexAssign) member function.
-	Expressions of the form $(D a[$(ARGUMENTS)] = c) are rewritten
-	as $(D a.opIndexAssign$(LPAREN)c,$(ARGUMENTS)$(RPAREN)).
+	Expressions of the form `a[`$(ARGUMENTS)`] = c` are rewritten
+	as `a.opIndexAssign$(LPAREN)c,` $(ARGUMENTS)`$(RPAREN)`.
```

Comment:
Before:

![image](https://user-images.githubusercontent.com/4370550/33967897-ba9705d8-e065-11e7-8461-7a619f4f34f4.png)

After:

![image](https://user-images.githubusercontent.com/4370550/33967903-bf8b5abc-e065-11e7-86e8-04fe8f8dbc2b.png)

Reply:
yah that was pretty clowny :)